presto-main/src/test/java/com/facebook/presto/execution/TestPlannerWarnings.java

```diff
@@ -96,8 +96,12 @@ public static void assertPlannerWarnings(LocalQueryRunner queryRunner, @Language
         WarningCollector warningCollector = new DefaultWarningCollector(new WarningCollectorConfig());
         try {
             queryRunner.inTransaction(sessionBuilder.build(), transactionSession -> {
-                rules.map(testingRules -> createPlan(queryRunner, transactionSession, sql, warningCollector, testingRules))
-                        .orElse(queryRunner.createPlan(transactionSession, sql, LogicalPlanner.Stage.CREATED, false, warningCollector));
```

Comment:
This could be switched to `orElseGet()`. But using an if/else seems cleaner since we are not using the value (we only compute it for its side effect), so I'd leave it as you have it.

Reply:
I noticed that only because IntelliJ highlighted "result of `.orElse` is not used". I know `orElseGet` would be technically enough, but this was simply surprising way of relying on side-effects.